Habitica/src/main/java/com/habitrpg/android/habitica/userpicture/UserPicture.java

```diff
@@ -1,29 +1,8 @@
 package com.habitrpg.android.habitica.userpicture;
 
-import com.magicmicky.habitrpgwrapper.lib.models.HabitRPGUser;
-import com.squareup.picasso.Picasso;
-import com.squareup.picasso.Target;
-
-import android.content.Context;
-import android.graphics.Bitmap;
-import android.graphics.BitmapFactory;
-import android.graphics.Canvas;
-import android.graphics.Paint;
-import android.graphics.Rect;
-import android.graphics.drawable.Drawable;
-import android.net.Uri;
-import android.util.Log;
-import android.widget.ImageView;
-
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.concurrent.atomic.AtomicInteger;
-
 public class UserPicture {
 
-    final protected static char[] hexArray = "0123456789ABCDEF".toCharArray();
+   /* final protected static char[] hexArray = "0123456789ABCDEF".toCharArray();
```

Comment:
Ok, i will fixed it. Should i made a new commit for fixes, or squash commit to last